tests/common/fixtures/ptfhost_utils.py

```diff
@@ -376,8 +379,8 @@ def run_garp_service(duthost, ptfhost, tbinfo, change_mac_addresses, request):
                 server_ipv4 = str(server_ipv4_base_addr + i)
                 server_ipv6 = str(server_ipv6_base_addr + i)
                 mux_cable_table[intf] = {}
-                mux_cable_table[intf]['server_ipv4'] = unicode(server_ipv4)
-                mux_cable_table[intf]['server_ipv6'] = unicode(server_ipv6)
+                mux_cable_table[intf]['server_ipv4'] = str(server_ipv4)
+                mux_cable_table[intf]['server_ipv6'] = str(server_ipv6)
```

Comment:
@nhe-NV Seems this change would cause failed related to python version of radv/test_radv_ipv6_ra.py in https://www.testbed-tools.org/scheduler/testplan/63f300c52824fb9bfa1073b5